ajax/libs/pizzicato/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "pizzicato",
+  "filename": "Pizzicato.min.js",
+  "description": "A web-audio library to simplify using and manipulating sounds.",
+  "version": "0.4.0",
+  "license": "MIT",
+  "homepage": "https://alemangui.github.io/pizzicato/",
+  "author": {
+    "name": "Alejandro Mantecon Guillen",
+    "email": "alemangui@gmail.com",
+    "url": "https://alemangui.github.io/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/alemangui/pizzicato.git"
+  },
+  "namespace": "Pizzicato",
+  "npmName": "pizzicato",
+  "npmFileMap": [
+    {
+      "basePath": "distr",
```

Comment:
@alemangui just to make sure if you are going to use `distr` in the npm packages in the future?